github/issues.go

```diff
@@ -250,10 +251,6 @@ func (s *IssuesService) Get(ctx context.Context, owner string, repo string, numb
 		return nil, nil, err
 	}
 
-	// TODO: remove custom Accept headers when APIs fully launch.
```

Comment:
Why are these custom accept headers being removed?
That seems to be an unrelated issue to #1458 and should likely be handled in a separate PR.

Also, the URL in the comment for this method needs updating, if you wouldn't mind:
`https://developer.github.com/v3/issues/#get-an-issue`

But again, maybe a separate PR would be more appropriate and keep this PR focused on the missing field as its description states, so please revert this change.